### ISSUE TITLE
"Module citeproc Error: Invalid string.." when using OSCOLA style

### ISSUE THREAD

**Issue by daniel-eder** [zepinglee/citeproc-lua#57]
**Describe the bug**
Document build fails with 
```
! Module citeproc Error: Invalid string..
\lua_now:e #1->\__lua_now:n {#1}
```

**Additional information**
- TeX distribution: MikTex 24.1 / MikTex Console 4.11 / LuaHBTeX 1.17.1
- Package `citation-style-language` version: 27.02.2024
- LaTeX engine: `lualatex`

**To Reproduce**
https://github.com/daniel-eder/reproduce-oscola-csl-error

1. Build using 2 consecutive calls lualatex to reproduce
2. Switch style to e.g. "harvard-cite-them-right" (line 20) and see that it works
3. I am not sure what exactly causes the error, *some* citations work with OSCOLA. See line 37-40. Citation on line 37 and 38 works, 39 and 40 do not. 

**Full Log**

```
Mode                 LastWriteTime         Length Name
----                 -------------         ------ ----
d-----        09.03.2024     15:26                build
This is LuaHBTeX, Version 1.17.1 (MiKTeX 24.1)
 restricted system commands enabled.
(./main.tex
LaTeX2e <2023-11-01> patch level 1
L3 programming layer <2024-02-20>
 (%AppData%/Local/Programs/MiKTeX/tex/latex/base/article.cls
Document Class: article 2023/05/17 v1.4n Standard LaTeX document class
(%AppData%/Local/Programs/MiKTeX/tex/latex/base/size12.clo))
(%AppData%/Local/Programs/MiKTeX/tex/latex/fontspec/fontspec.sty
(%AppData%/Local/Programs/MiKTeX/tex/latex/l3packages/xparse/xpars
e.sty
(%AppData%/Local/Programs/MiKTeX/tex/latex/l3kernel/expl3.sty
(%AppData%/Local/Programs/MiKTeX/tex/latex/l3backend/l3backend-lua
tex.def)))
(%AppData%/Local/Programs/MiKTeX/tex/latex/fontspec/fontspec-luate
x.sty (%AppData%/Local/Programs/MiKTeX/tex/latex/base/fontenc.sty)
 (%AppData%/Local/Programs/MiKTeX/tex/latex/fontspec/fontspec.cfg)
))
(%AppData%/Local/Programs/MiKTeX/tex/latex/geometry/geometry.sty
(%AppData%/Local/Programs/MiKTeX/tex/latex/graphics/keyval.sty)
(%AppData%/Local/Programs/MiKTeX/tex/generic/iftex/ifvtex.sty
(%AppData%/Local/Programs/MiKTeX/tex/generic/iftex/iftex.sty))
(%AppData%/Local/Programs/MiKTeX/tex/latex/geometry/geometry.cfg))
 (%AppData%/Local/Programs/MiKTeX/tex/latex/setspace/setspace.sty)
 (%AppData%/Local/Programs/MiKTeX/tex/generic/babel/babel.sty
(%AppData%/Local/Programs/MiKTeX/tex/generic/babel/luababel.def)
(%AppData%/Local/Programs/MiKTeX/tex/generic/babel/luababel.def)
*************************************
* Local config file bblopts.cfg used
*
(%AppData%/Local/Programs/MiKTeX/tex/latex/arabi/bblopts.cfg)
(%AppData%/Local/Programs/MiKTeX/tex/latex/babel-english/english.l
df))
(%AppData%/Local/Programs/MiKTeX/tex/generic/babel/locale/en/babel
-english.tex)
(%AppData%/Local/Programs/MiKTeX/tex/latex/citation-style-language
/citation-style-language.sty
(%AppData%/Local/Programs/MiKTeX/tex/latex/l3packages/l3keys2e/l3k
eys2e.sty) (%AppData%/Local/Programs/MiKTeX/tex/latex/url/url.sty)

(%AppData%/Local/Programs/MiKTeX/tex/latex/citation-style-language
/citation-style-language-init.sty)
(%AppData%/Local/Programs/MiKTeX/tex/latex/citation-style-language
/citation-style-language-data.sty)
(%AppData%/Local/Programs/MiKTeX/tex/latex/citation-style-language
/citation-style-language-cite.sty)
(%AppData%/Local/Programs/MiKTeX/tex/latex/citation-style-language
/citation-style-language-bib.sty)
(%AppData%/Local/Programs/MiKTeX/tex/latex/citation-style-language
/citation-style-language-compatible.sty))
No file main.aux.
(%AppData%/Local/Programs/MiKTeX/tex/latex/base/ts1cmr.fd)
*geometry* driver: auto-detecting
*geometry* detected driver: luatex
No file main.toc.
[1{%AppData%/Local/MiKTeX/fonts/map/pdftex/pdftex.map}]

! Module citeproc Error: Invalid string..
\lua_now:e #1->\__lua_now:n {#1}

l.39 ...1]{gorzalaEuropaischesHaftungssystemFur2023}
```

**Comment by zepinglee** [comment]
Thanks for your bug report and it's fixed in 022f606 .

**Comment by daniel-eder** [comment]
Awesome, thank you!
